### PR TITLE
Feature/todo

### PR DESCRIPTION
**Pull request by VanessaAoki**
In this branch, I implemented the basic structure of the todo application, using CRA and npm.

### CODE REVIEW

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-undef */
```
Comment (ghost) [inline src/components/TodoContainer.js:3]: You seem to have disabled many eslint rules, it would be great if you could consider actually refactoring the code as per linters instead since they are supposed to make our code more solid and clean. 

P.S.
disabling prefer-stateless-function is accepted since you are practising class components here

File: src/components/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable no-undef */
```
Comment (ghost) [inline src/components/TodoItem.js:4]: The same as above

File: src/components/TodoContainer.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          title: 'Develop website and add content',
+          completed: false,
+        },
+        {
+          id: 3,
+          title: 'Deploy to live server',
+          completed: false,
+        },
+      ],
+    };
```
Comment (ghost) [inline src/components/TodoContainer.js:29]: is this code actual? I mean is it being used by the app? I guess no since no sign of it was found in the UI. If this is the case please remove this one. This will ensure that your code is clean and solid.

In case this code is being used, I guess todos initial state is to be an empty array and we should add tasks in the form of objects dynamically.

**Review by ghost — COMMENTED** [review]
Hi @VanessaAoki  AWESOME PROJECT YOU GOT HERE :clap: :+1: 

**HIGHLIGHTS**
 
- All linters are passed :heavy_check_mark: 
- You documented your project in a professional way :books: 
- Good description of PR and commit messages :100: 
- The functions are behaving as expected :watch: 
- Your code is solid and meets all principals of DRY, KISS and YAGNI :+1: 

**SUGGESTIONS** 

- [ ] It would be great if you could include the linters config files into `.gitignore` file so to keep the root clean.
- [ ] _In addition, please check my inline comments, threat them as optional since they are just suggested to improve the quality of the code_ 

To sum up, you are doing a great job, and hope the above suggestions can serve as a basis to improve it at least by 0.1%.

Cheers! :sunglasses: :v:
